app/model/commands/PublishAtomCommand.scala

```diff
@@ -143,10 +146,23 @@ case class PublishAtomCommand(
 
     val publishedAtom = publishAtomToLive(updatedAtom)
     updateInactiveAssets(publishedAtom)
-
     publishedAtom
   }
 
+  private def sendWorldCupNotification(previewAtom: MediaAtom, maybePublishedAtom: Option[MediaAtom]) = {
+    val previewAtomHasTags = previewAtom.tags.contains("2018 world cup") || previewAtom.tags.contains("world cup 2018")
+
+    if (previewAtomHasTags) {
+      maybePublishedAtom match {
+        case Some(publishedAtom) => {
+          val hasNewAssets = hasNewAssets(previewAtom, publishedAtom)
```

Comment:
This won't compile, I believe, because it has the same name as the function. Also, it doesn't seem to be used?

Reply:
oops 😊! Updated!